public/app/features/alerting/unified/utils/misc.ts

```diff
@@ -102,6 +102,10 @@ export function makeDataSourceLink<T>(dataSource: DataSourceInstanceSettings<T>)
   return `${config.appSubUrl}/datasources/edit/${dataSource.uid}`;
 }
 
+export function makeDashboardLink(folderUID: string): string {
```

Comment:
shouldn't this be `makeFolderLink` ?

Reply:
Oh yea absolutely – `makeFolderLink` makes a lot more sense!